srcpkgs/rtl8192eu-dkms/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'rtl8192eu-dkms'
+pkgname=rtl8192eu-dkms
+version=20221031
+revision=1
+_modver=2022.10.31
+_gitrev=982f7907c478724b9fc4febd476b38c72dd4c1f2
+archs='*'
+wrksrc="rtl8192eu-linux-driver-${_gitrev}"
+depends="dkms bc"
+short_desc="Realtek 8192EU USB WiFi driver (DKMS)"
+maintainer="Phicem <phicem@gmx.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/Mange/rtl8192eu-linux-driver"
+distfiles="https://github.com/Mange/rtl8192eu-linux-driver/archive/${_gitrev}.tar.gz"
```

Comment:
if they don't have releases, then this package doesn't really meet our quality requirements

Reply:
Yes, you are right. I saw some exceptions in existing packages, including other realtek drivers like `rtl8812au-dkms`, `rtl8822bu-dkms`, and `rtl8821cu-dkms`, so I thought it was okay (and I still think that these are legitimate exceptions). But I should not say that the package meets Void quality requirements. I will edit my package presentation.

Reply:
[this upstream](https://github.com/clnhub/rtl8192eu-linux) has v5 of the driver and at least has branches for versions

Reply:
I switched to the suggested upstream. It seems to work equally well.